docs/content/docs/dev/table/sql/show.md

```diff
@@ -222,7 +222,7 @@ tEnv.executeSql("SHOW DATABASES").print()
 // create a table
 tEnv.executeSql("CREATE TABLE my_table (...) WITH (...)")
 // show tables
-tEnv.executeSql("SHOW TABLES").print()
+tEnv.executeSql("SHOW TABLES FROM default_database LIKE 'm%'").print()
```

Comment:
Touching this example may confuse users we only support the complex show tables syntax. Would be better to add an Example section in Show Tables (just like SHOW COLUMNS EXAMPLES) to list how to use the complex show table syntax.

Reply:
Detailed documentation is really very important, I have made an additional example like `SHOW COLUMNS EXAMPLES`, done :)